src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,54 +1,121 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './FindMovie.scss';
 
-// import { MovieCard } from '../MovieCard';
-
-export const FindMovie: React.FC = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import classNames from 'classnames';
+import { MovieCard } from '../MovieCard';
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+type Props = {
+  addMovie: (newMovie: Movie) => void
+};
+
+export const FindMovie: React.FC<Props> = (props) => {
+  const [movieTitle, setMovieTitle] = useState('');
+  const [isMovieFound, showMovie] = useState(true);
+  const [movie, setMovie] = useState<Movie | null>(null);
+
+  useEffect(() => {
+    showMovie(true);
+  }, [movieTitle]);
```

Comment:
why do you need useEffect here? you need to load a movie only by pressing a button